src/components/buttons/button.css

```diff
@@ -0,0 +1,7 @@
+.primary-button:hover {
+    box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+}
+
+span {
+    margin-right: 8px;
+}
```

Comment:
Isso vai fazer com que todos os span (quando esse arquivo for importado) tenham esse comportamento, né? Será que aqui não vale a pena também fazer um separadorzinho (só que dessa vez horizontal)?